src/features/query-bar/components/filter/ValueField.js

```diff
@@ -25,8 +26,9 @@ const getDateValue = selectedValues => {
     }
 
     const value = selectedValues[0];
-    if (value instanceof Date) {
-        return value;
+
+    if (Date.parse(String(value))) {
+        return new Date(value);
```

Comment:
The reason for the change here is that value was coming back as the string ("April 19, 2019") and not as a Date object which was causing an error.

The correct way is to first Date.parse the string to convert it to the number of milliseconds (as a string), and then return the Date object as that's what the DatePicker expects as the value prop.

Reply:
`Date.parse` looks like it returns different results based on implementation.
Do we use any libraries that validates dates?

Reply:
Will be reverting this change as it is not needed. It turns out formatDate was being used in the parent app incorrectly causing us to have to reformat it into a Date object.